src/content/docs/es/core-concepts/astro-components.mdx

```diff
@@ -286,11 +286,10 @@ const { titulo } = Astro.props;
 
 Los slots pueden ser transferidos a otros componentes. Por ejemplo, cuando se crean plantillas anidadas:
 
-```astro title="src/layouts/BaseLayout.astro" {10,13}
+```astro title="src/layouts/BaseLayout.astro" {9,12}
 // src/layouts/BaseLayout.astro
```

Comment:
I think in this case, this comment also needs removing:

```suggestion
```

Currently it looks like this: 
<img width="612" alt="code snippet with two titles and the wrong lines highlighted" src="https://github.com/withastro/docs/assets/357379/2d8ec258-ad2b-449d-ae15-02c6a6c29bf6">